fix filter branch filter

pmiossec opened the pull request:
by adding the name of the branch in the `git log` command arguments
that was accidentally removed during the refactoring to use ArgumentBuilder

fix #5278

Original code (notice the `{2}`):
https://github.com/gitextensions/gitextensions/blob/cd45f12f6356ae4127b962a91b8d6fc45885a32f/GitCommands/RevisionGraph.cs#L195

PS: @RussKie  with the release to come, I don't know if this fix should be done in master or release/3.00

RussKie commented:
If we're certain it is safe, send it to release/3.00. If not, send to
master.

On Tue, Nov 27, 2018, 11:35 PM Philippe Miossec <notifications@github.com>
wrote:

> by adding the name of the branch in the git log command arguments
> that was accidentally removed during the refactoring to use ArgumentBuilder
>
> fix #5278 <https://github.com/gitextensions/gitextensions/issues/5278>
>
> Original code (notice the {2}):
>
> gitextensions/GitCommands/RevisionGraph.cs
> <https://github.com/gitextensions/gitextensions/blob/cd45f12f6356ae4127b962a91b8d6fc45885a32f/GitCommands/RevisionGraph.cs#L195>
>
> Line 195 in cd45f12
> <http:///gitextensions/gitextensions/commit/cd45f12f6356ae4127b962a91b8d6fc45885a32f>
> "log -z {2} --pretty=format:\"{1}\" {0} {3} -- {4}",
>
> PS: @RussKie <https://github.com/RussKie> with the release to come, I
> don't know if this fix should be done in master or release/3.00
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/gitextensions/gitextensions/pull/5818
> Commit Summary
>
>    - fix filter branch filter
>
> File Changes
>
>    - *M* GitCommands/RevisionReader.cs
>    <https://github.com/gitextensions/gitextensions/pull/5818/files#diff-0>
>    (1)
>
> Patch Links:
>
>    - https://github.com/gitextensions/gitextensions/pull/5818.patch
>    - https://github.com/gitextensions/gitextensions/pull/5818.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/gitextensions/pull/5818>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/AEMyXrrh9DGU8ljqEYMtLt_9Ptdo1xTlks5uzTF7gaJpZM4Y1Zaw>
> .
>

RussKie commented:
Could you please rebase it on top of release/3.00 to make the build happy.

pmiossec commented:
That's when I rebased that it made the build unhappy :(
Anyway, rebased on the last commit done ;) We will see...

On Tue, 4 Dec 2018 at 11:50, RussKie <notifications@github.com> wrote:

> Could you please rebase it on top of release/3.00 to make the build happy.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/gitextensions/pull/5818#issuecomment-444057731>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAcFpESUpcZC9zsNWIFywf6ZT4BwvFoyks5u1lOLgaJpZM4Y1Zaw>
> .
>
